### ISSUE TITLE
How to print out predicted sequence in inference time?

### ISSUE THREAD

**Issue by tumbleintoyourheart** [ayumiymk/aster.pytorch#19]
Dear @ayumiymk,
Thank you for this repo,
In inference time, the script print out the accuracy, so I want it to print out also the actual predicted sequence. Is it stored in `lib/evaluators.py` `pred_list` variable?

**Comment by ayumiymk** [comment]
Yes, you are right!

**Comment by tumbleintoyourheart** [comment]
Thank you.

**Comment by tumbleintoyourheart** [comment]
In `main.py` there was an evaluation right before the training loop: `evaluator.evaluate()`.
So from your answer, the predicted sequence output at this step is supposed to be garbage, right?
(In my case it outputs all random Latin characters in a Japanese/Chinese text recognition problem)

I have done some Japanese/Chinese ocr with CRNN+CTC and the outputs at the very first steps were all blank characters (which is reasonable when using CTC loss).  When we are using the Attention decoder, is the above-mentioned phenomenon normal?

**Comment by ayumiymk** [comment]
For attention-based methods, it is normal to output random characters at the very first steps.

**Comment by tumbleintoyourheart** [comment]
I really appreciate the support. Thank you @ayumiymk.
